napari_arboretum/tree.py

```diff
@@ -7,6 +10,14 @@
 RED = np.array([255, 0, 0, 255], dtype=np.uint8)
 
 
+@dataclass
+class Edge:
+    x: Tuple[float, float]
+    y: Tuple[float, float]
+    color: Tuple[float, float, float, float]
+    id: Union[int, None] = None
```

Comment:
This is equivalent but the preferred way, I believe. Will need to import `Optional` from `typing`

```suggestion
    id: Optional[int] = None
```